pages/platform/ai/gi_07_manage_registry/guide.en-gb.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Use & manage existing and new image registries
+slug: use-manage-registries
+excerpt: Learn how to use and manage your public and private image registries
+section: General information
+order: 109
+updated: 2023-05-02
+---
+
+**Last updated 2nd May, 2023.**
+
+## Objective
+
+This guide is intended to help you **use and manage your public and private registries** through ovhai command line interface (AI CLI), API and OVHcloud control panel. This will help you when working with **OVHcloud AI Tools**, especially with **AI Training** and **AI Deploy**.
```

Comment:
```suggestion
This guide is intended to help you to **use and manage your public and private registries** through `ovhai` command line interface (AI CLI), API and [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB). This will help you when working with **OVHcloud AI Tools**, especially with **AI Training** and **AI Deploy**.
```